plugins/attachment-resources/src/components/AttachmentStyleBoxEditor.svelte

```diff
@@ -86,7 +86,10 @@
       haveUnsavedChanges = false
     }
 
-    await descriptionBox.createAttachments()
+    /*
+      Delayed save can be invoked after dismount. In this case descriptionBox would be null
+    */
+    await descriptionBox?.createAttachments()
```

Comment:
One more think about it, on dismount we need to force save instead to not loose state potentially.

Reply:
Thanks, added a handler for that too

Reply:
Nope, you need in case of haveUnsavedChanges, do clearTimeout and call save(saveObject, saveDescription)